cfme/fixtures/cli.py

```diff
@@ -18,25 +17,24 @@
 
 
 @contextmanager
-def fqdn_appliance(appliance, preconfigured, count):
-    version = appliance.version.vstring
-    stream = get_stream(appliance.version)
-    sprout_client = SproutClient.from_config()
+def fqdn_appliance(appliance, preconfigured, count, config):
+    sprout_client = SproutClient.from_config(sprout_user_key=config.option.sprout_user_key or None)
     apps, request_id = sprout_client.provision_appliances(
-        provider_type='rhevm', count=count, version=version, preconfigured=preconfigured,
-        stream=stream)
+        provider_type='rhevm',
+        count=count,
+        version=appliance.version.vstring,
+        preconfigured=preconfigured,
+        stream=appliance.version.stream(),
```

Comment:
Super small, but you don't need that comma here

Reply:
true, not needed, but when we enable `Black` it'll add it in these situations anyway:

`Black will add trailing commas to expressions that are split by comma where each element is on its own line. This includes function signatures.`

Reply:
its absolutely required for better and saner diffs on later changes